src/loop_libraries/library_fflite.cpp

```diff
@@ -0,0 +1,105 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+#include <limits>
+#include <boost/preprocessor/repeat.hpp>
+
+#include "library_fflite.hpp"
+#include "fflite.hpp"
+
+#define NAN_Q std::numeric_limits<double>::quiet_NaN()
+#define SET_TO_NAN(Z,INDEX,DATA) DATA[INDEX] = {NAN_Q, NAN_Q};
+#define UNDEFINED(R,ARGS,NAME) std::complex<double> Fflite::NAME ARGS noexcept\
+{\
+   return {NAN_Q, NAN_Q}; \
+}
+
+namespace looplibrary
+{
+
+Fflite::Fflite()
+{
+   ltini_();
+}
+
+std::complex<double> Fflite::A0(A_ARGS) noexcept
+{
+    std::complex<double> ca0;
+    int ier;
+    ljffca0_(ca0, 0, scl2_in, m02_in, ier);
+    return ca0;
+}
+
+std::complex<double> Fflite::B0(B_ARGS) noexcept
+{
+    std::complex<double> cb0;
+    int ier;
+    ljffcb0_(cb0, 0, scl2_in, p10_in, m02_in, m12_in, ier);
+    return cb0;
+}
+
+std::complex<double> Fflite::B1(B_ARGS) noexcept
```

Comment:
Please add a short description (maybe an equation) about how this calculation of `B1` works, because it is extremely in-transparent (partially because of the obscure function name of `FF`).

This applies to all other functions defined below, too.

Reply:
@Expander, these functions were defined before me in the ``pv.cpp`` file. What I did is just moving those definitions in the interface hoping that they have already been defined correctly. It means that I didn't get into details of fflite.